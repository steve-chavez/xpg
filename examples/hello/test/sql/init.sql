create extension hello;
\echo

select hello('world');
