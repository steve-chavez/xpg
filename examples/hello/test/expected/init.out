create extension hello;
\echo

select hello('world');
    hello    
-------------
 hello world
(1 row)

